ci/scripts/multi-arch-docker.sh

```diff
@@ -58,11 +52,20 @@ do
     pushGchr ${component} ${TAG}
   fi
 
-  if [ "${BUILDKITE_SOURCE}" == "schedule" ] || [ "${BUILDKITE_SOURCE}" == "ui" ]; then
-    # If this is a schedule/ui build, tag the image with the date.
+  if [ "${BUILDKITE_SOURCE}" == "schedule" ]; then
+    # If this is a schedule build, tag the image with the date.
     TAG="nightly-${date}"
     pushGchr ${component} ${TAG}
     pushDockerhub ${component} ${TAG}
+    TAG="latest"
```

Comment:
As we have discussed, latest should only point to release branch and only relased version should be pushed on dockerhub.

Reply:
Only push to GHCR here.